docs/source/cross_compile/config_vs_code.rst

```diff
@@ -264,9 +264,9 @@ documentation and `Configuring C/C++
 debugging <https://code.visualstudio.com/docs/cpp/launch-json-reference>`__.
 
 1. | Hit **Ctrl + Shift + P** to open the Command Palette, then search
-     for and select **Debug: Open launch.json.** Then, select **C/C++
-     (GDB)**. This will create a *launch.json* with the template
-     structure already present.
+     for and select **C/C++: Add Debug Configuration**. Then, select one of
+     the options - e.g., **(gdb) Launch**. This will create a *launch.json*
+     with the template structure already present.
```

Comment:
Can you also mention that this option is only present if you have a C/C++ file as the active file in the editor?